recipes/rnabob/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+mkdir -p $PREFIX/share/man/man1
```

Comment:
But this is not the location where you copy your file with the patch :)
You copy to $PREFIX/share/man/ which is fine imho.